src/Concerns/ManagesSubscriptions.php

```diff
@@ -20,13 +19,17 @@ public function newSubscription($name, $plan)
     }
 
     /**
-     * Get all of the subscriptions for the Paddle model.
+     * Get all of the subscriptions for the Billable model.
      *
-     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     * @return \Laravel\Paddle\Subscription[]|\Illuminate\Database\Eloquent\Collection
      */
     public function subscriptions()
     {
-        return $this->hasMany(Subscription::class, $this->getForeignKey())->orderByDesc('created_at');
+        if (is_null($this->customer)) {
+            return collect();
```

Comment:
Might be a dumb question, but if the user is doing any operations that are exclusive to Eloquent Collection, wouldn't this be an issue?

Reply:
Not a dumb question at all. I'll update this tomorrow 👍